portal/gatsby-node.js

```diff
@@ -139,7 +160,7 @@ exports.onCreateNode = ({ node, actions, getNode }) => {
     }
 };
 
-exports.createPages = async ({ graphql, actions, reporter }) => {
+exports.createPagesStatefully = async ({ graphql, actions, reporter }) => {
```

Comment:
![](https://media2.giphy.com/media/10Velb45AjmRos/giphy.gif?cid=ecf05e47k2qrda6w0u62x9zouzbhpv7g3prwq4izjsm0seth&rid=giphy.gif&ct=g)

Reply:
Jeg veksler mellom å være glad for å ha funnet denne og rasende for at det tilsynelatende bare var så enkelt som å skifte funksjon her og _hvorfor er det ikke åpenbart fra Gatsby-docs at dette er en Lur Ting å Bruke™_?

Kanskje jeg ombestemmer meg hvis vi oppdager at det var en god grunn til det, men fra testing lokalt så får den med seg endringer i `mdx` og assets (testet med nytt bilde) uten problemer, og uten å bygge _alle_ sider og knuse _alle bilder_ **hver gang**.

Reply:
På CI vil det ikke kutte ned byggtid på pull requests, siden `PREVIEW_PATH`. Men det burde ha noe å si for releaser på `main`. Og ikke minst på localhost.